### PR TITLE
[FIX] Fix pos_membership unmet dependency on membership_account_balance

### PR DESCRIPTION
**Pull request by franksongfeng**
add the module membership_account_balance

### CODE REVIEW

File: membership_account_balance/README.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Membership Management - Member Account Balance
```
Comment (elicoidal) [inline membership_account_balance/README.rst:1]: Please embed this into the standard README template

File: membership_account_balance/__init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline membership_account_balance/__init__.py:1]: Please remove and use standard short copyright notice
(date should be 2015)

File: membership_account_balance/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+#
+##############################################################################
+{'name': 'Membership Management - Member Account Balance',
+ 'version': '0.1',
+ 'category': '',
+ 'depends': ['account', 'membership', 'base_setup'],
+ 'author': 'Elico Corp',
+ 'license': 'AGPL-3',
+ 'website': 'https://www.elico-corp.com',
+ 'description': """
```
Comment (elicoidal) [inline membership_account_balance/__openerp__.py]: remove: redundant with README

File: membership_account_balance/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+Total membership is calculated in the customer.
+
+Contributors
+------------
+
+* Alex Duan <alex.duan@elico-corp.com>
+* Siyuan Gu: <gu.siyuan@elico-corp.com>
+""",
+ 'images': [],
+ 'demo': [],
```
Comment (elicoidal) [inline membership_account_balance/__openerp__.py]: remove demo and test keys

File: membership_account_balance/README.rst [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+-----
+A new check box records the membership for the account.
+(Default is not membership)
+
+Total membership is calculated in the customer.
+
+Contributors
+------------
+
+* Alex Duan <alex.duan@elico-corp.com>
```
Comment (elicoidal) [inline membership_account_balance/README.rst:26]: add my name and contact information

**Review by elicoidal — CHANGES_REQUESTED** [review]
some light modifications to follow OCA format.
Not checked the code and not needed (module is working fine in production)

File: membership_account_balance/__openerp__.py (hunk 4 above)
Comment (franksongfeng) [reply membership_account_balance/__openerp__.py]: done.

File: membership_account_balance/__openerp__.py (hunk 3 above)
Comment (franksongfeng) [reply membership_account_balance/__openerp__.py]: done.

File: membership_account_balance/__init__.py (hunk 2 above)
Comment (franksongfeng) [reply membership_account_balance/__init__.py:1]: done.

File: membership_account_balance/README.rst (hunk 1 above)
Comment (franksongfeng) [reply membership_account_balance/README.rst:1]: done.

File: membership_account_balance/README.rst (hunk 5 above)
Comment (franksongfeng) [reply membership_account_balance/README.rst:26]: done.

File: membership_account_balance/__init__.py [hunk 6]
```diff
@@ -1,22 +1,4 @@
...
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2015 Elico Corp (www.elico-corp.com).
```
Comment (martintorresvm) [inline membership_account_balance/__init__.py:2]: @franksongfeng @elicoidal If we modify this information should we update the current year (2017)? . And the same for the changed headers.

File: membership_account_balance/__init__.py (hunk 2 above)
Comment (franksongfeng) [reply membership_account_balance/__init__.py:1]: @elicoidal travis has 2 main issues: The module membership_accont_balance has deprecated syntax, Would I change its code?; Other modules except membership_... has many warning/errors,  How to handle them?

Comment (elicoidal) [reply membership_account_balance/__init__.py:1]: ignore warning (non blocking).
Deprecated syntax might not be a problem: check if this is blocking.

Anyway I fear the travis for the repo v8.0 might have many other issues.

**Comment by elicoidal** [comment]
No keep 2015 which is the date we created the code and for which we can claim ownership

**Comment by elicoidal** [comment]
@franksongfeng any improvement on this one?

File: membership_account_balance/__init__.py (hunk 6 above)
Comment (franksongfeng) [reply membership_account_balance/__init__.py:2]: the latest copyright year is needed to confirm by elico, @elicoidal, can we use 2017 as the latest year?

**Comment by elicoidal** [comment]
2015 is the copyright year.
 




------------------
--
 	    Eric Caudal [Founder and CEO]
                              Skype: elico.corp. Phone: + 86 186 2136 1670 (Cell), + 86 21 6211 8017/27/37 (Office)             
             Elico Shanghai (Hong Kong/Shenzhen/Singapore)             
Odoo Gold Partner // Best Odoo Partner APAC 2014 and 2016
 

 
 
 
------------------ Original ------------------
From: "Frank Song (Elico)"; 
Date: Thursday, Jan 12, 2017 8:34 PM
To: "Elico-Corp/odoo-addons"; 
Cc: "Eric Caudal"; "Mention"; 
Subject: Re: [Elico-Corp/odoo-addons] [FIX] Fix pos_membership unmetdependency on membership_account_balance (#140)

 

@franksongfeng commented on this pull request.
 
 
In membership_account_balance/__init__.py:
 > -# -#    This program is free software: you can redistribute it and/or modify -#    it under the terms of the GNU Affero General Public License as -#    published by the Free Software Foundation, either version 3 of the -#    License, or (at your option) any later version. -# -#    This program is distributed in the hope that it will be useful, -#    but WITHOUT ANY WARRANTY; without even the implied warranty of -#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the -#    GNU Affero General Public License for more details. -# -#    You should have received a copy of the GNU Affero General Public License -#    along with this program.  If not, see <." target="_blank">http://www.gnu.org/licenses/>. -# -############################################################################## +# © 2015 Elico Corp (www.elico-corp.com).  
the latest copyright year is needed to confirm by elico, @elicoidal, can we use 2017 as the latest year?

**Comment by seb-elico** [comment]
> @franksongfeng any improvement on this one?

@elicoidal is there some more things that Frank needs to fix? Seems like all your previous comments have been addressed.

**Comment by franksongfeng** [comment]
seb,
now my one PR (https://github.com/Elico-Corp/odoo-addons/pull/140)  is not merged, this can caused some modules  including pos_membership failed.  If any issue still existed after merging #140, that will be puzzled for me.
now my testbed including pr #140 work well.

--
Frank Song [Developer]
                  Skype: fowlersong. Phone:                   + 86 186 2155 9600 (Cell), + 86 21 6211 8017/27/37                   (Office) 
                 Elico Shanghai                   (Hong Kong/Shenzhen/Singapore) 
Odoo Gold Partner,                     best Odoo Partner 2016 for APAC 

 
 
 
------------------ Original ------------------

File: membership_account_balance/README.rst (hunk 1 above)
Comment (elicoidal) [reply membership_account_balance/README.rst:1]: If I am not wrong this is not the standard README template

**Comment by elicoidal** [comment]
This PR seems doesnot exit

File: membership_account_balance/README.rst (hunk 1 above)
Comment (elicoidal) [reply membership_account_balance/README.rst:1]: @seb-elico @xuan139 Can you please male sure this is finished once and for all?

**Comment by seb-elico** [comment]
@franksongfeng looks like a duplicate of #45, please make sure you avoid duplicates!
@noahzaozao IMO we should close this PR, please advise

**Comment by elicoidal** [comment]
Closing for duplicated

**Pull request closed without merging**